package.json

```diff
@@ -154,6 +154,16 @@
                         "replace",
                         "allowDuplicates"
                     ]
+                },
+                "numberedBookmarks.fillColor": {
+                    "type": "string",
+                    "default": "#00ff25",
+                    "description": "Bookmark background color"
+                },
+                "numberedBookmarks.numberColor": {
```

Comment:
Rename the setting to `gutterIconNumberColor` to be more clear that this _numberColor_ is related to the gutter icon. It also be similar to the other `gutterIconPath` from my other Bookmarks extension [link](
https://github.com/alefragnani/vscode-bookmarks/#available-settings)